### DISCUSSION TITLE
Is this Yi Dome Guard supported

### DISCUSSION THREAD

**Discussion by witzker** [roleoroleo/yi-hack-Allwinner-v2#657] [Q&A]
Hi, I'm totally new to this stuff!
What I have here is a:

![grafik](https://github.com/roleoroleo/yi-hack-Allwinner-v2/assets/4120266/91c0f960-eb78-48cc-b364-f1857f1d1781)

I have connected it to the original App, there is the firmware after update shows:
12.1.05.1_202302091435

Is this device supported to get it into Synology with ONVIF?

Hope for help to achieve this.
Manx THX

**Comment by roleoroleo** [comment]
There are 2 main problems.
- Firmware 12.1.05 is not supported at the moment. If you make a backup of the flash I can try to support it.
- But the the answer to the "Synology with ONVIF" is "I don't know", because ONVIF is not a good standard (IMHO). You should try.

**Reply by witzker** [reply]
Many THX for taking care and your offer!
@: If you make a backup of the flash I can try to support it.

As said I'm totally new with this stuff, so can you Pls. tell me How-To make this backup if the flash.

**Reply by roleoroleo** [reply]
Check this wiki:
https://github.com/roleoroleo/yi-hack-Allwinner-v2/wiki/Dump-your-backup-firmware-(SD-card)

**Comment by ma8883** [comment]
Today, both cameras, the same as those of the topicstarter, are no longer available via IP, only available in YIHome. Is there any point in flashing now? Is firmware 12.1.05.1_202302091435 supported?

**Comment by ma8883** [comment]
I can’t get an image from this camera through Yi Home Cameras with yi-hack?

**Comment by roleoroleo** [comment]
> Today, both cameras, the same as those of the topicstarter, are no longer available via IP, only available in YIHome. Is there any point in flashing now? Is firmware 12.1.05.1_202302091435 supported?

Yes 12.0.05 is supported but if you apply a fw updatede from yi app when the cam is already hacked, the behavior is unpredictable.
Yoou should remove the hack and apply it again.

**Comment by witzker** [comment]
> Yes 12.0.05 is supported 

Do you need an Image for this as said above or is it OK Now?
If Yes where can I get it?
Many THX for taking care!

**Reply by roleoroleo** [reply]
No.
12.1.05 is supported.
Download the released image for r30gb.

**Comment by witzker** [comment]
My intention is to get this cam into Synology Surveillance Station (supporting ONVIF)
What I saw here still problems:
https://github.com/roleoroleo/yi-hack-Allwinner-v2/issues/484
How can I help ?

**Comment by iagoampro** [comment]
Hola, tengo un problema ya que no puedo flashear la cámara. Es la primera vez que intento algo así, asi que pido disculpas por mi torpeza por adelantado. A continucación comento las características técnicas y lo que he hecho esperando así alguna ayuda. 

**Vídeo tutorial que vi**:
- https://www.youtube.com/watch?v=P7zTmwF5FjM 

**Cámara:** 
- Yi Dome Guard YRS 3019
- Firmware 12.1.05.1_202302091435
- https://www.yitechnology.com/dome-R37/ 

**Pasos**
- He formateado mi tarjeta SD de 64gb a FAT32

- He descargado, descomprimido y copiado los archivos necesarios a la tarjeta SD desde este link https://github.com/alienatedsec/yi-hack-v5/releases/tag/0.3.8. He probado todos por separado. 
![image](https://github.com/roleoroleo/yi-hack-Allwinner-v2/assets/143079712/23e4d2be-4699-4f35-a13a-85693de0d82a)

- He conectado la cámara a la aplicación YiHome funcionando correctamente

- He desenchufado la cámara y he insertado la tarjeta SD con los archivos

- He vuelto a enchufar la cámara, gira, parpadea una luz roja y se vuelve color verde fija

- Navego a mi ip/:8080 pero no encuentra nada y en la aplicación YiHome funciona todo correctamente.

**Comment by roleoroleo** [comment]
Why you donwloaded the file from yi-hack-v5 repository and opened an issue here?
Based on your fw revision, your cam is a r30gb.
You have to download the proper file from this repo and follow the instruction in the readme.

**Reply by iagoampro** [reply]
@roleoroleo no tengo demasiados conocimientos técnicos (aunque sí ganas de adquirirlos) He descargado cosas incorrectas para probar y he escrito en este hilo porque fué a donde llegué. Pido disculpas por mi ignorancia. 
Leyendo varias preguntas y respuestas he llegado a esa conclusión, necesito los archivos con el prefijo r30gb, el caso es que no sé de donde descargarlos. Puede spor favor indicarme con un link el lugar correcto para la descarga e instalación? 
Muchísimas gracias por lo que haces y por tu rapida respuesta, que tengas muy buen día

**Reply by roleoroleo** [reply]
Download the file for your cam here:
https://github.com/roleoroleo/yi-hack-Allwinner-v2/releases/tag/0.3.1

And follow the installation guide here:
https://github.com/roleoroleo/yi-hack-Allwinner-v2

**Reply by iagoampro** [reply]
Muchísimas gracias @roleoroleo , eres muy amable. Mañana cuando vuelva al la oficina lo pruebo y hago una pequeña donación por tu trabajo.
